cypress/integration/tests/developer/controls/stakeholdergroups/filter.test.ts

```diff
@@ -102,7 +102,7 @@ describe("Stakeholder groups filter validations", { tags: "@tier2" }, function (
         }
 
         // Clear all filters
-        clickByText(button, clearAllFilters);
+        clickByText(button, clearAllFilters, true);
```

Comment:
This is not needed , by default isforced in clickbytext method is set to true.

Reply:
Done